src/components/Discussion/Internal/docs.md

```diff
@@ -6,6 +6,26 @@ Components on this page are not exported from the styleguide. This documentation
 
 # Comment
 
+#### Link Preview
+
+_TODO: put further down that list_
```

Comment:
rm or do it

Reply:
I'm not sure what it means…

Reply:
I think she wanted to move it down in the doc file – first show the more common cases – e.g. move it after the body.